libraries/SITL/SIM_Vicon.cpp

```diff
@@ -77,7 +77,7 @@ void Vicon::update_vicon_position_estimate(const Location &loc,
     // calculate a random time offset to the time sent in the message
     // simulates a time difference between the remote computer and autopilot
     if (time_offset_us == 0) {
-        time_offset_us = (unsigned(random()) % 7000) * 1000000ULL;
+        time_offset_us = (unsigned(random()) % 7000) * 1000ULL;
```

Comment:
Do we really want to change this?  I think this is making the timing of the messages much more even... perhaps to the point that there is almost no jitter in the timing of the messages?

Reply:
Before my change it was 7,000 seconds of jitter... instead of what I think was originally desired to be 7,000 milli-seconds

Reply:
Ah wait.. this is for holding jitter, this is for holding a time difference between the external computer and the autopilot.  This is only set once and shouldn't have any impact on performance because AP should learn the offset.

What happens when you remove this?  Does it still work OK?  If yes then I think we should leave this unchanged.

Reply:
Oh shoot sorry! I misunderstood the purpose of this one! The time difference part made me think jitter instead of system clock differences. Should I change the comment to "simulates a difference in system clocks between the remote computer and autopilot"?

Reply:
It already has this comment, "// simulates a time difference between the remote computer and autopilot" so I'd just backout the change.  If you want to update the comment to make it more clear that's also OK.  I don't mind either way.